skopt/searchcv.py

```diff
@@ -271,12 +271,13 @@ class BayesSearchCV(BaseSearchCV):
 
     def __init__(self, estimator, search_spaces, optimizer_kwargs=None,
                  n_iter=50, scoring=None, fit_params=None, n_jobs=1,
-                 iid=True, refit=True, cv=None, verbose=0,
+                 n_points=1, iid=True, refit=True, cv=None, verbose=0,
```

Comment:
Can you add `n_points` to the doc string and modify the `n_jobs` docs so that people know what they both do and how they interact?